src/common/sentry-init.ts

```diff
@@ -0,0 +1,21 @@
+import * as Sentry from '@sentry/react';
+import { Integrations } from '@sentry/tracing';
+
+import { userHasAllowedDiagnosticsKey } from '@store/onboarding/onboarding.hooks';
```

Comment:
should probs be in a common/constants file, not a hook file

Reply:
it only concerns the key of the storage hook, so think its okay to leave here